flink-core/src/main/java/org/apache/flink/api/common/eventtime/WatermarkStrategy.java

```diff
@@ -49,4 +53,158 @@
 	 */
 	@Override
 	WatermarkGenerator<T> createWatermarkGenerator(WatermarkGeneratorSupplier.Context context);
+
+	/**
+	 * Creates a watermark strategy for situations with monotonously ascending timestamps.
+	 *
+	 * <p>The watermarks are generated periodically and tightly follow the latest
+	 * timestamp in the data. The delay introduced by this strategy is mainly the periodic interval
+	 * in which the watermarks are generated.
+	 *
+	 * @see AscendingTimestampsWatermarks
+	 */
+	static <T> WatermarkStrategy<T> forMonotonousTimestamps() {
```

Comment:
Shouldn't this be `public` and not `package-private`? (The same for the other methods)

Reply:
it's an interface

Reply:
True, I commented while only looking at the diff, not the whole class :(